workloads/hyper-scale/common.sh

```diff
@@ -40,7 +40,21 @@ prep(){
 
 setup(){
     export MGMT_CLUSTER_NAME=$(oc get infrastructure cluster -o jsonpath='{.status.infrastructureName}'| cut -c 1-13)
-    export BASEDOMAIN=$(oc get dns cluster -o jsonpath='{.spec.baseDomain}')
+    export MGMT_BASEDOMAIN=$(oc get dns cluster -o jsonpath='{.spec.baseDomain}')
+    export MGMT_AWS_HZ_ID=$(aws route53 list-hosted-zones | jq -r '.HostedZones[] | select(.Name=="'${MGMT_BASEDOMAIN}'.")' | jq -r '.Id')
+    if [[ $HC_EXTERNAL_DNS != "false" ]]; then
```

Comment:
Nit but can we just say if its true lol

Reply:
Also, we should check the openshift version here as well to make sure we arent attempting w/ a < 4.12 image